adoc/deployment-sysreqs.adoc

```diff
@@ -156,6 +156,30 @@ The management workstation needs at least the following networking permissions:
 * Access to the `apiserver` (the load balancer should expose it, port `6443`), that will in turn talk to any master in the cluster
 * Access to Dex on the configured `NodePort` (the load balancer should expose it, port `32000`) so when the OIDC token has expired, `kubectl` can request a new token using the refresh token
 
+==== Sub-Network Sizing
+
+[IMPORTANT]
+====
+The service subnet and pod subnet must not overlap.
+
+Please plan generously for workload and the expected size of the networks before bootstrapping.
+====
+
+The default pod subnet is `10.244.0.0/16`. It allows for 65536 IP addresses overall.
+Assignment of CIDR's is by default `/24` (254 usable IP addresses per node).
+
+The default node allocation of `/24` means a hard cluster node limit of 254 since this is the number of `/24` ranges that fit in a `/16` range.
```

Comment:
Is this a `hard cluster node limit` or a pod limit? Isnt those /24 addresses given to the pods *in* the node, so thats a maximum of 254 pods per node?

* not an expert in networking, disregard and shame me if wrong

Reply:
@ereslibre Could you clarify this for me please.

Reply:
The pod subnet is a cluster-wide limit. Now, if you choose the default: `10.244.0.0/16`, then you can partition that subnet as desired -- the total number of available addresses will be 65536 --. Every node will get assigned a CIDR of its own by Kubernetes. By default it's a `/24` -- check `--node-cidr-mask-size` in the controller manager -- (what allows for 254 IP addresses per node).

Following that reasoning, if every node gets a `/24` when registering, then you can fit 256 nodes in total on a `10.244.0.0/16`.

Depending on the planning of the cluster they might want to change the CIDR allocation for each node (fitting more nodes, less IP addresses per node, or the other way around), or change the global pod subnet altogether.